plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -65,8 +64,9 @@ class DbtConfigArgs:
     profiles_dir: Optional[str] = None
     profile: Optional[str] = None
     target: Optional[str] = None
+    threads: int = 1
     single_threaded: bool = False
-    vars: str = ""
+    vars: dict = None
```

Comment:
Nit: This should be `Dict`, not `dict`. I think support for the latter was only added in Python 3.10.